src/TextField/EnhancedTextarea.js

```diff
@@ -56,9 +56,15 @@ class EnhancedTextarea extends Component {
   };
 
   state = {
-    height: this.props.rows * rowsHeight,
+    height: rowsHeight,
```

Comment:
Good question, `null` has the advantage of saving 1 bytes over `undefined` once minified.